interface_service/requirements.txt

```diff
@@ -6,4 +6,6 @@ pytest
 requests
 pylint==2.12.2
 dynaconf
-jinja2
\ No newline at end of file
+jinja2
+google-cloud-storage
```

Comment:
Since we don't need GCS library anymore, it can be removed from the requirements. Also, idk why but MinIO client is missing and should be here.

Btw - add the MinIO library with constant version:
```
minio==7.1.8
```